SplunkRumCrashReporting/SplunkRumCrashReporting/CrashReporting.swift

```diff
@@ -84,7 +132,9 @@ func loadPendingCrashReport(_ data: Data!) throws {
         exceptionType = report.exceptionInfo.exceptionName
     }
 
-    let oldSessionId = String(decoding: report.customData, as: UTF8.self)
+    guard let dict = dataToDictionary(data: report.customData) else {return}
```

Comment:
We should make a best effort to report data, even if we are missing some fields.  Missing the old session ID and/or the old screen name should not prevent us from reporting any crash reporting span.